ddtrace/tracer/tracer.go

```diff
@@ -317,17 +377,22 @@ func (t *tracer) Extract(carrier interface{}) (ddtrace.SpanContext, error) {
 
 // flush will push any currently buffered traces to the server.
 func (t *tracer) flushPayload() {
+	defer func(start time.Time) {
+		t.config.statsd.Timing("datadog.tracer.flush.duration", time.Since(start), nil, 1)
+	}(time.Now())
```

Comment:
Is it really a flush if there were no items?
Maybe this should go after the check for itemCount?

Reply:
👍 Moved.